### PR TITLE
Test image builds for ARM64 arch in CI

### PR DESCRIPTION
**Pull request by sivanantha321**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/transformer-docker-publish.yml [hunk 1]
```diff
@@ -27,15 +27,18 @@ jobs:
...
-          if [ -f docker-compose.test.yml ]; then
-            docker-compose --file docker-compose.test.yml build
-            docker-compose --file docker-compose.test.yml run sut
-          else
-            cd python
-            docker buildx build . --file custom_transformer.Dockerfile
-          fi
+        uses: docker/build-push-action@v5
+        with:
+          platforms: linux/amd64
```
Comment (spolti) [inline .github/workflows/transformer-docker-publish.yml:36]: some places there still only amd64, is it due compatibility with dependencies for arm?

Comment (sivanantha321) [reply .github/workflows/transformer-docker-publish.yml:36]: That is for custom model and transformer which is only servers as an example. We will have to add ARM support for huggingface in a separate PR.

Comment (yuzisun) [reply .github/workflows/transformer-docker-publish.yml:36]: Any issue with the ARM build for huggingface server?

Comment (sivanantha321) [reply .github/workflows/transformer-docker-publish.yml:36]: Yeah
```
#8 0.202 exec /bin/sh: exec format error
#8 ERROR: process "/bin/sh -c apt-get update -y && apt-get install python3.10-venv -y && apt-get clean &&     rm -rf /var/lib/apt/lists/*" did not complete successfully: exit code: 1

#5 [builder  1/13] FROM docker.io/nvidia/cuda:12.1.0-devel-ubuntu22.04@sha256:e3a8f7b933e77ecee74731198a2a5483e965b585cea2660675cf4bb152237e9b
#5 CANCELED
------
 > [prod 2/7] RUN apt-get update -y && apt-get install python3.10-venv -y && apt-get clean &&     rm -rf /var/lib/apt/lists/*:
0.202 exec /bin/sh: exec format error
------
huggingface_server.Dockerfile:39

```

Comment (sivanantha321) [reply .github/workflows/transformer-docker-publish.yml:36]: nvm, my docker builder is not configured properly.

Comment (sivanantha321) [reply .github/workflows/transformer-docker-publish.yml:36]: Looks like vllm does not provides arm64 wheels

**Review by spolti — APPROVED** [review]
/lgtm

**Comment by sivanantha321** [comment]
@yuzisun Can you verify auto merge works with this change ?

**Comment by yuzisun** [comment]
> @yuzisun Can you verify auto merge works with this change ?

Sure, what was the fix?

**Comment by sivanantha321** [comment]
> > @yuzisun Can you verify auto merge works with this change ?
> 
> Sure, what was the fix?

There is not any explicit fix. Those custom transformer workflows uses actions now instead of manual push. I just wanted to know if it is affects the auto merge.

**Comment by yuzisun** [comment]
> > > @yuzisun Can you verify auto merge works with this change ?
> > 
> > 
> > Sure, what was the fix?
> 
> There is not any explicit fix. Those custom transformer workflows uses actions now instead of manual push. I just wanted to know if it is affects the auto merge.

seems like it did not work

**Pull request merged**